### PR TITLE
Standardizing the number of updates

### PR DESCRIPTION
**Pull request by Xavrax**

### CODE REVIEW

File: world/GameManager.cpp [hunk 1]
```diff
@@ -47,11 +47,20 @@ void GameManager::addObject(std::unique_ptr<IObject> object, const std::string&
 }
 
 void GameManager::gameLoop() {
+    auto timestamp = 60;
```
Comment (ThudPoland) [inline world/GameManager.cpp]: Magic number?

File: world/GameTimers.cpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+void GameTimers::measureElapsedTime() {
+    _timeSinceLastUpdate += _gameClock.restart();
+}
+
+const sf::Time &GameTimers::timeSinceLastUpdate() const {
+    return _timeSinceLastUpdate;
+}
+
+void GameTimers::synchronizeWithTimestamp(sf::Time timeForFrame) {
```
Comment (ThudPoland) [inline world/GameTimers.cpp]: const sf::Time&

File: world/GameTimers.hpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#ifndef GAMETIMERS_HPP
+#define GAMETIMERS_HPP
+
+#include <SFML/System/Clock.hpp>
+
+class GameTimers {
+private:
```
Comment (ThudPoland) [inline world/GameTimers.hpp]: Please move private below public.

File: world/GameManager.cpp [hunk 4]
```diff
@@ -47,11 +47,20 @@ void GameManager::addObject(std::unique_ptr<IObject> object, const std::string&
...
+    auto timePerFrame = sf::seconds(1.f / timestamp);
```
Comment (Kicer86) [inline world/GameManager.cpp]: Would std::chrono be applicable here?

File: world/GameTimers.hpp (hunk 3 above)
Comment (Xavrax) [reply world/GameTimers.hpp]: reference to: https://github.com/MIPSOXPL/Abandoned-Ancestry/blob/master/world/GameManager.hpp

It's yours, isn't it?

File: world/GameManager.cpp (hunk 1 above)
Comment (Xavrax) [reply world/GameManager.cpp]: dont have any config file so i used template value

File: world/GameManager.cpp (hunk 4 above)
Comment (Xavrax) [reply world/GameManager.cpp]: Let's give it a try

File: world/GameTimers.cpp (hunk 2 above)
Comment (Xavrax) [reply world/GameTimers.cpp]: k

File: world/GameManager.cpp (hunk 1 above)
Comment (ThudPoland) [reply world/GameManager.cpp]: For now we can save some const in code.

File: world/GameTimers.hpp (hunk 3 above)
Comment (ThudPoland) [reply world/GameTimers.hpp]: Will fix this. :)

**Pull request merged**